webapp-django/crashstats/settings/base.py

```diff
@@ -359,9 +373,6 @@ def path(*dirs):
 )
 MANAGERS = ADMINS
 
-# import logging
-# LOGGING = dict(loggers=dict(playdoh={'level': logging.DEBUG}))
-
```

Comment:
This was commented out, playdoh-related, and we don't need it anymore. So nixing it.